test-case-reporting/src/db_interfacing.ts

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2020 The AMP HTML Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Build, DB, Job, TestCase, TestRun} from 'test-case-reporting';
+import {Database, TIMESTAMP_PRECISION} from './db';
+
+// Types in the DB namespace interface use snake_case instead of camelCase.
+/* eslint @typescript-eslint/camelcase: "off" */
+function getDateFromTimestamp(timestamp: number): Date {
+  const msConversionConstant: number = Math.pow(10, 3 - TIMESTAMP_PRECISION);
```

Comment:
Lets define this once up top

Reply:
What does "up top" mean? In the top of the file, or?

Reply:
What do you mean by "up top"? In the top of the file, or...?

Reply:
yep, let's make it a file level constant

Reply:
I believe it is already in the top of the file; it's before the class declaration.

Reply:
Oh, gotcha.
Done.